packages/js/email-editor/src/private-apis/index.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import {
+	// @ts-expect-error No types for privateApis.
+	privateApis as blockEditorPrivateApi,
+	// @ts-expect-error No types for privateApis.
+	privateApis as blockEditorPrivateApis,
+} from '@wordpress/block-editor';
```

Comment:
There is a single `privateApis` [export](https://github.com/WordPress/gutenberg/blob/9b0e2d96679ab6c82033946bead9521529e6b29a/packages/block-editor/src/index.js#L23). You can simplify this section.

Reply:
Nice catch.  This is not intentional.

Reply:
You need to update this too https://github.com/mailpoet/mailpoet/blob/1bf95f4b0a554e275006b24940e5a7e21a7cae31/packages/js/email-editor/src/private-apis/index.ts#L62

from `blockEditorPrivateApi`
to `blockEditorPrivateApis`